File: 03.02.2025/Enum for Days with Typedef.c
#include <stdio.h>

typedef enum {
    SUN = 1, MON, TUE, WED, THU, FRI, SAT
} Weekday;

int main() {
    Weekday day = TUE;
    printf("Day number: %d\n", day);  // Outputs: 3
    return 0;
}

File: 03.02.2025/Enum for Days with Typedef.py
from enum import Enum

class Weekday(Enum):
    SUN = 1
    MON = 2
    TUE = 3
    WED = 4
    THU = 5
    FRI = 6
    SAT = 7

day = Weekday.TUE
print(f"Day number: {day.value}")
